stdlib/public/SwiftRemoteMirror/CMakeLists.txt

```diff
@@ -1,15 +1,29 @@
 # libswiftRemoteMirror.dylib should not have runtime dependencies; it's
 # always built as a shared library.
 if(SWIFT_BUILD_DYNAMIC_STDLIB OR SWIFT_BUILD_REMOTE_MIRROR)
-  add_swift_target_library(swiftRemoteMirror
-                    SHARED TARGET_LIBRARY DONT_EMBED_BITCODE NOSWIFTRT
-                    SwiftRemoteMirror.cpp
-                    LINK_LIBRARIES
-                      swiftReflection
-                    C_COMPILE_FLAGS
-                      ${SWIFT_RUNTIME_CXX_FLAGS} -DswiftRemoteMirror_EXPORTS
-                    LINK_FLAGS
-                      ${SWIFT_RUNTIME_LINK_FLAGS}
-                    INSTALL_IN_COMPONENT
-                      swift-remote-mirror)
+  if (SWIFT_BUILD_STDLIB)
+    add_swift_target_library(swiftRemoteMirror
+                      SHARED TARGET_LIBRARY DONT_EMBED_BITCODE NOSWIFTRT
```

Comment:
Why not conditional-ize these single options instead of duplicating the whole `add_swift_target_library` call.

Something like:

```CMake
set(remote_mirror_options STATIC)
if(SWIFT_BUILD_STDLIB)
  set(remote_mirror_options SHARED TARGET_LIBRARY DONT_EMBED_BITCODE NOSWIFTRT)
endif()

add_swift_target_library(swiftRemoteMirror
  ${remote_mirror_options}
  SwiftRemoteMirror.cpp
  LINK_LIBRARIES
    swiftReflection
  C_COMPILE_FLAGS
    ${SWIFT_RUNTIME_CXX_FLAGS} -DswiftRemoteMirror_EXPORTS
  LINK_FLAGS
    ${SWIFT_RUNTIME_LINK_FLAGS}
  INSTALL_IN_COMPONENT
    swift-remote-mirror)
```